packages/features/ee/users/components/UsersTable.tsx

```diff
@@ -76,6 +76,31 @@ function UsersTableBare() {
     },
   });
 
+  const lockUserAccount = trpc.viewer.admin.lockUserAccount.useMutation({
+    onSuccess: ({ userId, locked }) => {
+      showToast(locked ? "User was locked" : "User was unlocked", "success");
```

Comment:
nit: translation?

Reply:
It's an admin-only view. It can wait IMO